### PR TITLE
Add more event hooks for conversation view

### PR DESCRIPTION
**Pull request by ben181231**
connect #103
connect #148

### CODE REVIEW

File: SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift [hunk 1]
```diff
@@ -1557,18 +1582,39 @@ extension SKYChatConversationViewController {
...
+    }
+}
+
 extension SKYChatConversationViewController {
 
+    open func subscribeToPubsubConnectivity() {
+        self.skygear.pubsub.delegate = self
+    }
+
+    open func unsubscribeToPubsubConnectivity() {
```
Comment (cheungpat) [inline SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift]: `unsubscribeFromPubsubConnectivity`

File: SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift [hunk 2]
```diff
@@ -92,6 +92,16 @@ import SKPhotoBrowser
     @objc optional func messageStatusShouldShowInConversationViewController(
         _ controller: SKYChatConversationViewController) -> Bool
 
+    /**
+     * Hooks on pubsub connectivity
+     */
+
+    @objc optional func pubsubDidConnectedInConversationViewController(
```
Comment (cheungpat) [inline SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift]: s/DidConnected/DidConnect/g

File: SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift [hunk 3]
```diff
@@ -92,6 +92,16 @@ import SKPhotoBrowser
...
+        _ controller: SKYChatConversationViewController)
+
+    @objc optional func pubsubDidDisconnectedInConversationViewController(
```
Comment (cheungpat) [inline SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift]: s/DidDisconnected/DidDisconnect/g

**Review by cheungpat — CHANGES_REQUESTED** [review]

**Comment by ben181231** [comment]
Updated.

**Pull request merged**